### DISCUSSION TITLE
Updating ebsd unit cell after changes to spatial data

### DISCUSSION THREAD

**Discussion by jhiscocks** [mtex-toolbox/mtex#565] [Ask Anything] [answered]
Starting with an EBSD data set, say we want to scale it, and shift it slightly in x (see below).  
```
ebsd.x=ebsd.x*0.9108+0.0255;
```
The changes now need to be propagated to the ebsd.unitCell  property.  Previously I did that using the following command:
```
ebsd.unitCell = calcUnitCell([ebsd.prop.x(:),ebsd.prop.y(:)]);
```
However, it does not seem to make any changes to the ebsd.unitCell property.  Is this update still necessary?  Is it made a different way? How do I check it's been implemented?

Thanks. Jessica.

**Comment by ralfHielscher** [comment, marked as answer]
Hi Jessica,

the unit cell is automatically updated. No need to to this yourself.

Ralf.

**Reply by jhiscocks** [reply]
Good to know.  I think it was different in previous versions but I'm glad we can skip that step now.

Cheers, J
